code/modules/mob/mob.dm

```diff
@@ -1929,7 +1930,11 @@ mob/proc/on_foot()
 		alpha = lowest_alpha
 
 /mob/proc/teleport_to(var/atom/A)
+	var/last_slip_value = can_slip
+	unlock_from()
```

Comment:
move this back to the wiz code, we dont want everything to unlock itself if it attempts a teleport